archetypes/case-studies.md

```diff
@@ -3,7 +3,8 @@ title: "{{ replaceRE "^[-0-9]{10}-" "" .Name | replaceRE "-" " " | title }}"
 slug: "{{ replaceRE "^[-0-9]{10}-" "" .Name }}"
 headertext: Case Studies
 subheadertext: "Real Results: AI in Action"
-problem: "Describe client's problem here"
+industry: Add client's industry here
```

Comment:
We will use the industry and service to filter case studies in the future.

Reply:
Awesome!